### PR TITLE
MVP met

### PR DESCRIPTION
**Pull request by briannakeune**
I was able to complete the assignment up until 14.

I also left the TK challenges in here as well. 

My only real blocker was installing the pipenv and running it's shell in the file itself.

### CODE REVIEW

File: src/08_comprehensions.py [hunk 1]
```diff
@@ -10,14 +10,14 @@
...
+y = [x + 1 for x in range(5)]
 
-print (y)
+print(y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
-y = []
+y = [x * x for x in range(9)]
```
Comment (Brodt7258) [inline src/08_comprehensions.py:20]: this is a good spot for python's exponent syntax. So, x ** 3. You're good though.

File: src/14_cal.py [hunk 2]
```diff
@@ -29,4 +29,50 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
+
+def get_calendar(args):
```
Comment (Brodt7258) [inline src/14_cal.py:35]: This is really well-done. While you weren't _required_ to validate input, it's great that you did. And knowing when and how to break problems up into smaller sub-functions is going to be really useful.

**Review by Brodt7258 — APPROVED** [review]
I saw you got the linter up and running too. Those are so useful in any language, but I find them nearly-required for Python. Definitely a good choice, and I'll credit you for stretch goals.

File: src/08_comprehensions.py (hunk 1 above)
Comment (briannakeune) [reply src/08_comprehensions.py:20]: Oohhh. I was dumb and my brain kept reading it as squared rather than cubed. But thank you!

**Pull request merged**